cli.js

```diff
@@ -59,15 +61,22 @@ try {
 	throw error;
 }
 
-const appInfo = plist.parse(infoPlist);
-const appName = appInfo.CFBundleDisplayName || appInfo.CFBundleName;
-const appIconName = appInfo.CFBundleIconFile.replace(/\.icns/, '');
-const dmgPath = path.join(destPath, `${appName} ${appInfo.CFBundleShortVersionString}.dmg`);
-
 const ora = new Ora('Creating DMG');
 ora.start();
 
 async function init() {
+	let appInfo;
+	if (infoPlist[0] === '<') {
```

Comment:
This test feels a bit fragile. What if there's whitespace before the content? I assume `plist.parse` throws when trying to parse binary plist, so you could just do a try catch.

Reply:
Fixed ✅